backend/src/test/kotlin/no/nav/mulighetsrommet/api/kafka/KafkaTest.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.mulighetsrommet.api.kafka
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.maps.shouldContainValues
+import io.mockk.clearMocks
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import no.nav.mulighetsrommet.api.domain.Tiltakskode
+import no.nav.mulighetsrommet.api.domain.Tiltakstype
+import no.nav.mulighetsrommet.api.services.TiltakstypeService
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+class KafkaTest : FunSpec({
+
+    val topicMap = mapOf(Pair("tiltakendret", "topic-tiltakendret"))
+    val mockTiltakstypeService = mockk<TiltakstypeService>()
+    val eventProcessor = EventProcessor(topicMap, mockTiltakstypeService)
+    val kafkaFactory = KafkaFactory(mockTiltakstypeService)
+
+    afterTest {
+        clearMocks(mockTiltakstypeService)
+    }
+
+    context("KafkaFactory") {
+        test("should return the correct topic map") {
```

Comment:
Som du var inne på så er jeg enig i at denne testen ikke har så mye for seg, men den andre ser da fornuftig ut